NitroxServer/Serialization/World/WorldManager.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Windows;
+using NitroxModel.Helper;
+using NitroxModel.Server;
+
+namespace NitroxServer.Serialization.World;
+
+public static class WorldManager
+{
+    public static readonly string SavesFolderDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Nitrox", "saves");
+
+    private static readonly List<Listing> savesCache = new();
+
+    static WorldManager()
+    {
+        try
+        {
+            Directory.CreateDirectory(SavesFolderDir);
+        }
+        catch (Exception ex)
+        {
+            MessageBox.Show(ex.ToString(), "Error", MessageBoxButton.OK, MessageBoxImage.Error);
+            return;
```

Comment:
NitroxServer is not a UI project and should not refernce `MessageBox`. Let this error bubble up to the calling code and handle it there (NitroxLauncher).